app/assets/stylesheets/core/footer.scss

```diff
@@ -55,7 +59,9 @@
   }
 
   &Extras {
-    padding: $size-0 114px 66px 114px;
+    max-width: 1200px;
+    margin: 0 auto;
+    padding: $size-0 34px 66px;
```

Comment:
Out of curiosity, would we want this section to have the same padding for smaller screen breakpoints?

Reply:
Mm actually good point. Going to reduce mobile horizontal padding here to stay consistent with the static content container.